facade/src/main/java/org/jboss/pnc/facade/providers/BuildProviderImpl.java

```diff
@@ -67,20 +69,27 @@
     private BuildConfigurationAuditedRepository buildConfigurationAuditedRepository;
     private Gerrit gerrit;
     private BuildConfigurationRevisionMapper buildConfigurationRevisionMapper;
+    private BuildMapper buildMapper;
+
+    private BuildCoordinator buildCoordinator;
 
     @Inject
     public BuildProviderImpl(BuildRecordRepository repository,
                              BuildMapper mapper,
                              BuildConfigurationRepository buildConfigurationRepository,
                              BuildConfigurationAuditedRepository buildConfigurationAuditedRepository,
                              Gerrit gerrit,
-                             BuildConfigurationRevisionMapper buildConfigurationRevisionMapper) {
+                             BuildConfigurationRevisionMapper buildConfigurationRevisionMapper,
+                             BuildMapper buildMapper,
```

Comment:
`BuildMapper` is here twice.

Reply:
I remove the second argument. Good catch! :)